client/src/styles/global.css

```diff
@@ -0,0 +1,53 @@
+
+/* global.css */
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+:root {
+  --cyf-color: #ff1e1e;
+  --white: #ffffff;
+  --gray: #808080;
+  --black: #000000:
+}
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  color: var(--black);
+  background-color: var(--white);
+}
+
+a {
+  color: blue;
+  text-decoration: none;
+}
+
+h1 {
+  font-size: 24px;
+}
+
+table{
+  width: 88%;
+  border-collapse: collapse;
+}
+
+th {
+  border: 1px solid var(--gray);
+  background-color: var(--cyf-color  );
+  padding: 5px;
+}
+
+td {
+  border: 1px solid var(--gray);
+  text-align: center;
+  background-color: var(--white);
+  padding: 10px;
+}
+
+h2 {
+	font-weight: 100;
+  margin: 10px;
+}
+  /* Add h1  and h3 */
```

Comment:
We discussed using the font sizes from CYF main site here for h1 h2 h3

It would be helpful to group the heading together I see h1 at the top of this file.

We also discussed removing our overwriting of margin from global css

Reply:
@MohammadSefidgar Please take a look at this